src/Type/NeverType.php

```diff
@@ -334,12 +333,12 @@ public function shuffleArray(): Type
 
 	public function isCallable(): TrinaryLogic
 	{
-		return TrinaryLogic::createYes();
+		return TrinaryLogic::createNo();
 	}
 
 	public function getCallableParametersAcceptors(ClassMemberAccessAnswerer $scope): array
 	{
-		return [new TrivialParametersAcceptor()];
+		return [];
```

Comment:
This should `throw new ShouldNotHappenException();` instead

Reply:
hmm it throws..

```
1) Error
The data provider specified for PHPStan\Analyser\NodeScopeResolverTest::testFileAsserts is invalid.
PHPStan\ShouldNotHappenException: Internal error.
/Users/staabm/workspace/phpstan-src/src/Type/NeverType.php:341
/Users/staabm/workspace/phpstan-src/src/Analyser/NodeScopeResolver.php:4361
/Users/staabm/workspace/phpstan-src/src/Analyser/NodeScopeResolver.php:4107
/Users/staabm/workspace/phpstan-src/src/Analyser/NodeScopeResolver.php:4560
/Users/staabm/workspace/phpstan-src/src/Analyser/NodeScopeResolver.php:2320
/Users/staabm/workspace/phpstan-src/src/Analyser/NodeScopeResolver.php:4619
/Users/staabm/workspace/phpstan-src/src/Analyser/NodeScopeResolver.php:2320
/Users/staabm/workspace/phpstan-src/src/Analyser/NodeScopeResolver.php:770
/Users/staabm/workspace/phpstan-src/src/Analyser/NodeScopeResolver.php:304
/Users/staabm/workspace/phpstan-src/src/Testing/TypeInferenceTestCase.php:95
/Users/staabm/workspace/phpstan-src/src/Testing/TypeInferenceTestCase.php:154
/Users/staabm/workspace/phpstan-src/src/Testing/TypeInferenceTestCase.php:292
/Users/staabm/workspace/phpstan-src/tests/PHPStan/Analyser/NodeScopeResolverTest.php:17

```